extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/configproperties/ConfigPropertiesBuildStep.java

```diff
@@ -35,17 +26,26 @@
 
 public class ConfigPropertiesBuildStep {
 
+    @BuildStep
+    void produceConfigPropertiesMetadata(CombinedIndexBuildItem combinedIndex,
```

Comment:
This is a really minor complaint, but the rest of the class doesn't use lambda's so can we keep it consistent please?

Reply:
OK will do. But I still think that streams are more readable :)

Reply:
I used to think the same :). Let's keep the streams in the other uses since those are for new classes

Reply:
Used to? What changed?

Reply:
Disillusionment :). I liked them because they were cool, then once I stepped back from them for a while I started disliking them and find that I need to think more about what they are doing

Reply:
I sort of agree with simple cases, like the one in this method. Both stream and loop are easy to read in this case. But if it would be a method with a bunch of it/else and loops, I don't think it would more readable than stream with lambdas.

Reply:
I agree with you there